src/Lykke.AzureStorage/Tables/Paging/PagingInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace Lykke.AzureStorage.Tables.Paging
+{
+    public class PagingInfo
+    {
+        public PagingInfo()
+        {
+            PreviousPages = new List<string>();
+        }
+        /// <summary>
+        /// specify null to fetch all records
+        /// </summary>
+        public int? ElementCount { get; set; }
```

Comment:
Should all properties be writeable?

Reply:
Actually, at least half of them are being accessed and changed during the object lifetime. It's just simple DTO object, so, why not.